src/zenml/orchestrators/local/local_orchestrator.py

```diff
@@ -87,6 +96,7 @@ def prepare_or_run_pipeline(
             string_utils.get_human_readable_time(run_duration),
         )
         self._orchestrator_run_id = None
+        return None
```

Comment:
I have seen this in a few more places. Why do we explicitly return a `None` here?

Reply:
Because `mypy` wants it that way 😄
But more generally, this returns an optional submission result, so it only makes sense to return one of the two options: `None` or `SubmissionResult(...)`